changelogs/fragments/128-remove_tests_sanity_requirements.yml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
+  - sanity tests - remove unnecessary requirements.txt file (https://github.com/ansible-collections/overview/issues/43).
```

Comment:
Same here.

Reply:
otherwise it will fail, required for the collection